src/simulators/matrix_product_state/matrix_product_state_internal.hpp

```diff
@@ -228,6 +228,21 @@ class MPS{
 
   void get_probabilities_vector(rvector_t& probvector, const reg_t &qubits) const;
 
+//----------------------------------------------------------------
+  // Function name: get_prob_single_qubit_internal
+  // Description: Returns the probability of measuring outcome 0 (or 1)
+  //   for a single qubit in the standard basis.
+  //   It does the same as get_probabilities_vector but is faster for
+  //   a single qubit, and is used during measurement.
+  // Parameters: qubit - the qubit for which we want the probability
+  //             outcome - probability for 0 or 1
+  //             mat - the matrix computed for the probability. This
+  //                   may be useful for further computations.
```

Comment:
I know that it's difficult but it can be good if you can find a way to provide more details about `mat`

Reply:
done.